charts/stacks/observability/values.yaml

```diff
@@ -67,11 +56,6 @@ elastic:
         count: 3
         config:
           node.roles: ["data_warm", "ingest", "ml", "transform"]
-          xpack.security.authc:
```

Comment:
@rsevilla87 you also need to delete lines 99-103 for this file

Reply:
good catch, fixed!